### PR TITLE
Require trailing commas for multiline blocks

### PR DESCRIPTION
**Pull request by viart**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +18,10 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "babel-eslint": "^6.0.0",
-    "eslint": "^3.0.1",
-    "estraverse-fb": "^1.3.1"
```
Comment (viart) [inline package.json]: Was it used somewhere?

**Comment by edvinerikson** [comment]
Looks good to me

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "codestyle",
-  "version": "9.0.0",
+  "version": "10.0.0",
```
Comment (edvinerikson) [inline package.json]: This shouldn't be bumped in the PR, right?

Comment (viart) [inline package.json]: No, it should be bumped in PR.

**Comment by EnzoMartin** [comment]
So is this discussion dead?

**Comment by viart** [comment]
rebased

File: README.md [hunk 3]
```diff
@@ -1,7 +1,25 @@
 # Codestyle
 
-Uses eslint `2.x`
+Uses eslint `3.x`
 
 ```
-npm install codestyle
+npm install codestyle --save
```
Comment (edvinerikson) [inline README.md]: save-dev maybe?

Comment (viart) [inline README.md]: done

**Pull request merged**